src/components /Sighting.js

```diff
@@ -0,0 +1,48 @@
+import React, { useState, useEffect, startTransition } from "react";
+import axios from "axios";
+import { serverURL } from "../ServerURL";
+import { useParams, Link } from "react-router-dom";
+import { Card } from "antd";
+
+const Sighting = () => {
+  const [sightings, setSightings] = useState([]);
+  const [sightingIndex, setSightingIndex] = useState(0);
+  const params = useParams();
+  console.log(params.sightingIndex);
+
+  useEffect(() => {
+    axios
+      .get(`${serverURL}/${sightingIndex}`)
+      .then((response) => setSightings(response.data));
+  }, [sightingIndex]);
+
+  if (sightingIndex != params.sightingIndex)
+    setSightingIndex(params.sightingIndex);
```

Comment:
Why use a state for sightingIndex? Seems a bit redundant to me here and we could just use the param instead.